tests/tests/cpld_regional_natl_c192

```diff
@@ -0,0 +1,72 @@
+###############################################################################
+#
+#  FV3 CCPP regional c192 test for North Atlantic domain
```

Comment:
I think we agreed to remove the `CCPP` in the test files, because there is no other option anymore.

Reply:
@climbfuji, Removed accordingly. Also please note the new added HAFS regression test names have been updated. Thanks!